ml-data-hub-plugin/src/main/groovy/com/marklogic/gradle/task/MigrateProvenanceTask.groovy

```diff
@@ -0,0 +1,13 @@
+package com.marklogic.gradle.task
+
+import com.marklogic.hub.provenance.ProvenanceManager
+
+class MigrateProvenanceTask extends AbstractConfirmableHubTask {
```

Comment:
Should this be added to DataHubPlugin, or not yet?

Reply:
Added it to the plugin. If we're uncomfortable with the task being visible in 5.6 we can comment it out later.

Reply:
Yeah I'm good either way - I can see some value in users being able to do this so they can try out e.g. /v1/graphs/things on the new provenance data.